fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -117,6 +117,30 @@ public Message getMessage(final FedoraEvent jcrEvent,
         return message;
     }
 
+    private String getUserData(final FedoraEvent jcrEvent) {
```

Comment:
By this, I assume you are suggesting rethrowing `javax.jcr.RepositoryException` as another, `org.fcrepo` exception?